web/temp_views.py

```
@@ -0,0 +1,54 @@
+@login_required
```

Ruff (0.8.2)

Undefined name `login_required`

(F821)